pkg/init/Dockerfile

```diff
@@ -1,9 +1,17 @@
-FROM alpine:3.5
+FROM linuxkit/alpine:c5021a113803f7608e3908014d316c3490183d0b@sha256:4c7a80173c71ca5019dc56f40442b3a8345f141dd46593bd1fe6d130294d07cb AS build
+RUN apk add \
+    bsd-compat-headers \
+    gcc \
+    linux-headers \
+    musl-dev
+COPY ./start-stop-daemon.c .
 
-RUN \
-  apk --no-cache update && \
-  apk --no-cache upgrade -a && \
-  apk --no-cache add \
-  && rm -rf /var/cache/apk/*
+RUN cc -static -O2 -Wall -Werror -o start-stop-daemon start-stop-daemon.c
```

Comment:
I can do that now. `linuxkit/init` is relatively easy as it has no further dependencies. `linuxkit/tini` is a pain as it triggers a chain of update.